src/scryptTypes.ts

```diff
@@ -9,15 +9,14 @@ export class ScryptType {
   protected _literal: string;
   protected _asm: string;
   protected _type: string;
-  protected _finalType: string;
+  public static alias: AliasEntity[] = [];
```

Comment:
Better keep `_finalType` field bcoz it makes senson, `alias` should not be a member of ScryptType.

Reply:
refactor: using _typeResolver
18e4b4264b4ac49eb31218dad09ba0283c7cb83b